src/data_management/rasters/NewRaster2.js

```diff
@@ -26,6 +27,19 @@ class NewRaster2 extends Component {
   render() {
     return (
       <ManagementForm onSubmit={onSubmitExample} wizardStyle>
+        <Slushbucket
+          name="sharedWithOrganisation"
+          title="Shared With Organisation"
+          choices={[
+            { value: "abc", display: "aBc" },
+            { value: "nens", display: "nEns" },
+            { value: "home", display: "hOmbre" },
+          ]}
+          readonly={false}
+          isFetching={false}
```

Comment:
isFetching kan weg.

Reply:
We zullen ipv isFetching het formulier pas tonen nadat alle relevante data binnen is